<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileManagerDialogBase</class>
 <widget class="QDialog" name="ProfileManagerDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profile Manager</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_profiles">
      <attribute name="title">
       <string>Profiles</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="createProfileButton">
            <property name="text">
             <string>New profile</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="copyProfileButton">
            <property name="text">
             <string>Copy profile</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editProfileButton">
            <property name="text">
             <string>Rename profile</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeProfileButton">
            <property name="text">
             <string>Remove profile</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Profiles</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="list_profiles"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_datasources">
      <attribute name="title">
       <string>Import</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Source profile</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxNamesSource"/>
          </item>
          <item>
           <widget class="QLabel" name="choseProfileLabel">
            <property name="text">
             <string>Target profile</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxNamesTarget"/>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_checkAll">
            <property name="toolTip">
             <string>Select all items in the source profile</string>
            </property>
            <property name="text">
             <string>Select all items</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeSourcesButton">
            <property name="toolTip">
             <string>Remove selected items from source profile</string>
            </property>
            <property name="text">
             <string>Remove selected items</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="importButton">
            <property name="toolTip">
             <string>Import selected items from source to target profile</string>
            </property>
            <property name="text">
             <string>Import selected items</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_inner_datasources">
          <attribute name="title">
           <string>Data Sources</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QTreeWidget" name="treeWidgetSource">
             <column>
              <property name="text">
               <string>Source Profile:</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QTreeWidget" name="treeWidgetTarget">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
             <column>
              <property name="text">
               <string>Target Profile:</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_plugins">
          <attribute name="title">
           <string>Plugins</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Plugins in source profile</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="list_plugins"/>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Plugins in target profile</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="list_plugins_target">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_models">
          <attribute name="title">
           <string>Other (Import only)</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QFrame" name="frame_3">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QCheckBox" name="bookmark_check">
                <property name="text">
                 <string>Bookmarks</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="favourites_check">
                <property name="text">
                 <string>Favourites</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="models_check">
                <property name="text">
                 <string>Models</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="scripts_check">
                <property name="text">
                 <string>Scripts</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="styles_check">
                <property name="text">
                 <string>Styles + Labels</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="ui_check">
                <property name="text">
                 <string>UI Customization (e.g. hidden toolbar icons)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="functions_check">
                <property name="text">
                 <string>Expression Functions</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_qdt_export">
      <attribute name="title">
       <string>QDT Export</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="qdt_profile_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Profile</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="qdt_qgis_min_version_label">
         <property name="text">
          <string>QGIS min. version</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="qdt_description_label">
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QComboBox" name="qdt_export_profile_cbx"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="qdt_email_label">
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="qdt_email_edit"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="qdt_version_edit"/>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QCheckBox" name="qdt_clear_export_folder_checkbox">
         <property name="text">
          <string>Clear export folder</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="qdt_qgis_min_version_edit"/>
       </item>
       <item row="11" column="0" colspan="3">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QCheckBox" name="qdt_inactive_plugin_export_checkbox">
         <property name="text">
          <string>Export inactive plugins</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="qdt_version_label">
         <property name="text">
          <string>Version</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="3">
        <widget class="QPushButton" name="export_qdt_button">
         <property name="text">
          <string>Export</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="qdt_qgis_max_version_label">
         <property name="text">
          <string>QGIS max. version</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="qdt_qgis_max_version_edit"/>
       </item>
       <item row="2" column="1">
        <widget class="QTextEdit" name="qdt_description_edit"/>
       </item>
       <item row="0" column="1">
        <widget class="QgsFileWidget" name="qdt_file_widget">
         <property name="storageMode">
          <enum>QgsFileWidget::GetDirectory</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_export_folder">
         <property name="text">
          <string>Export folder</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="closeDialog">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
